script/prerender/prerender.tsx

```diff
@@ -73,7 +74,10 @@ async function render() {
 
   await renderSitemapIndex();
   await renderManifest();
-  await createRedirects(archiveLoader, osWebLoader);
+
+  if (APP_ENV === 'production') {
+    await createRedirects(archiveLoader, osWebLoader);
+  }
```

Comment:
Since I've added bunch of files to `/data/redirects/` this script is trying now create redirects for them and it fails because we use fixtures files for archive and os web loaders

We can either do the `if` like I did or we can change `books` variable in `createRedirects` function to an empty array in test env. Another option would be to create mock file in `data/redirects/` and change this `books` variable to `['testbook1-uuid.json']`

I don't think it really matters

Reply:
Actually, it may be good to make sure that `createRedirects` is run properly in CI. I'll make sure it is

Reply:
resolved with c25b347